### ISSUE TITLE
Add an option to disable sending nick changes to server buffers

### ISSUE THREAD

**Issue by BlacklightShining** [weechat/weechat#308]
Sure, they can be hidden using `/filter`, but it isn't easy to also prevent them from being written to the server logs.

**Comment by Shawn-Smith** [comment]
If you turn the logger level down lower than 2 it wont log them.

**Comment by BlacklightShining** [comment]
@Shawn-Smith And it also won't log things like mode changes and all those numerics that get sent on connect, which I _do_ want logged.

**Comment by Shawn-Smith** [comment]
Maybe logger should be changed to be a string instead of a number so you can define exactly what is being logged.

Like:
````
logger.level.irc.freenode = user_message,user_notice,user_private,mode_change
logger.level.irc.freenode.#spam = none
logger.level.irc.someother = user_message,user_notice,user_private
````

Or something of that nature.

**Comment by BlacklightShining** [comment]
Maybe it could be a list of tags. Users could specify lists like `log1,log2,log3` to keep old behavior, and I could use `*,!irc_nick` for server buffers.

**Comment by stfnm** [comment]
I agree, being able to specify a list of tags what to log or not to log would be great!

**Comment by flashcode** [comment]
Waiting for an option, it's easy to disable such messages on server buffer with a trigger (so they are not displayed and logged at all):

    /trigger add server_no_nick_changes modifier "weechat_print" "${type} == server && ${tg_tags} =~ ,irc_nick," "/.*//"
